homeassistant/components/cambridge_audio/entity.py

```diff
@@ -0,0 +1,27 @@
+"""Base class for Cambridge Audio entities."""
+
+from aiostreammagic import StreamMagicClient
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class CambridgeAudioEntity(Entity):
+    """Defines a base Cambridge Audio entity."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
What other platforms will you add? You might want to move the `_attr_name = None` to the media player, since the other platforms you will add probably won't have the device name as the entity name

Reply:
Yep, good catch. I'll be adding an update entity and a few switch/button entities.